features/put_filter_outputs.feature_put_filter_outputs_needs_fixing

```diff
@@ -49,9 +47,7 @@ Feature: Put Filter Outputs Private Endpoints Enabled
       },
       {
         "id": "94310d8d-72d6-492a-bc30-27584627edb2",
-        "filter_id": "94310d8d-72d6-492a-bc30-27584627fil2",
-        "instance_id": "94310d8d-72d6-492a-bc30-27584627inst2",
-        "state": "completed",
+        "state": "string",
```

Comment:
It would be good to use a more real **state**, such as `completed` or `published`